nilearn/interfaces/fmriprep/load_confounds_utils.py

```diff
@@ -10,13 +10,14 @@
 from nilearn._utils.fmriprep_confounds import (
     _flag_single_gifti, _is_camel_case
 )
+from nilearn.interfaces.bids import parse_bids_filename
 
 
 img_file_patterns = {
     "aroma": "_desc-smoothAROMAnonaggr_bold",
-    "nii.gz": "_space-.*_desc-preproc_bold.nii.gz",
-    "dtseries.nii": "_space-.*_bold.dtseries.nii",
-    "func.gii": "_space-.*_hemi-[LR]_bold.func.gii",
+    "nii.gz": "(_space-.*)?_desc-preproc_bold\.nii\.gz",
```

Comment:
Just curious: what is the effects of `\` in the string ?

Reply:
I think it escapes the . as a wildcard character in regular expressions